runtime/vara/src/weights/pallet_balances.rs

```diff
@@ -88,37 +88,37 @@ impl<T: frame_system::Config> pallet_balances::WeightInfo for SubstrateWeight<T>
 // For backwards compatibility and tests
 impl WeightInfo for () {
     fn transfer() -> Weight {
-        Weight::from_ref_time(55_241_000 as u64)
+        Weight::from_ref_time(84_789_000 as u64)
             .saturating_add(RocksDbWeight::get().reads(1 as u64))
             .saturating_add(RocksDbWeight::get().writes(1 as u64))
     }
     fn transfer_keep_alive() -> Weight {
-        Weight::from_ref_time(40_852_000 as u64)
+        Weight::from_ref_time(62_856_000 as u64)
```

Comment:
why all these increased?

Reply:
Possibly due to substrate update. The last time I did not update the weights.

Reply:
When I made my investigations with benchmarks I'd 2x dispersion for the same benchmark for the same core number, but different time of measurement.